test/test.jl

```diff
@@ -1,16 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
```

Comment:
Yes, I have, because I was tired of waiting 10 minutes for `test linalg/triangular` when I wanted to test `diagonal`.